{"uuid": "a1f6ce73-f805-4b0a-9c57-c15f15bab1a8", "children": ["a06a9bd0-87a3-44f0-adde-af933932f6b6"], "befores": [{"name": "take_screenshot_when_failure", "status": "passed", "start": 1559914946679, "stop": 1559914946680}], "afters": [{"name": "take_screenshot_when_failure::1", "status": "passed", "start": 1559914946687, "stop": 1559914946687}, {"name": "take_screenshot_when_failure::tear_down", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/allure_commons/_allure.py\", line 169, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/ace/Documents/GitHub/epam_ta/pom/conftest.py\", line 25, in tear_down\n    if request.node.rep_call.failed:\n"}, "start": 1559914946687, "stop": 1559914946689}], "start": 1559914946679, "stop": 1559914946689}